app/styles/atoms/button.less

```diff
@@ -1,22 +1,35 @@
 .generate-extended-palette-btn(@variant) {
-  @outline-color: ~'--color-@{variant}-50';
-
-  .outline(var(~'--color-@{variant}-50'));
-  .fill(var(~'--color-@{variant}-50'));
+  background-color: var(~'--color-@{variant}-50');
+  border: 1px solid var(~'--color-@{variant}-500');
   color: var(~'--color-@{variant}-500');
-  transition: all 0.2s ease;
+
+  i,
+  span {
+    color: var(~'--color-@{variant}-500');
+  }
+
+  a& {
+    line-height: 2.4rem;
+  }
```

Comment:
unfortunately we have to keep these as there are still some places where we used buttons styles on `a` tags (on some `LinkTo` for example)